client/services/FeedbackGeneratorServiceSpec.js

```diff
@@ -149,6 +202,71 @@ describe('FeedbackGeneratorService', function() {
     });
   });
 
+ /*
+  TODO: Having a hard time getting coverage for getFeedback else branch
```

Comment:
I am trying to achieve coverage for the else branch of the getFeedback function, but was having a hard time having it not error out. I removed it for now.